pyproject.toml

```diff
@@ -12,7 +12,9 @@ classifiers = [
     "Topic :: Software Development :: Libraries :: Python Modules",
     "Topic :: System :: Networking"
 ]
-exclude = ["tests/*"]
+packages = [
+    { include = "roombapy" },
+]
```

Comment:
Does this effectively exclude tests still?

Reply:
I think so, I checked my own libraries and they never exclude tests but they have this, _so I'm assuming this does it implicitly_